### PR TITLE
Feature/incremental count

### PR DESCRIPTION
**Pull request by n13**
Incremental counting for rep and cbs scores.

### CODE REVIEW

File: src/seeds.accounts.cpp [hunk 1]
```diff
@@ -542,19 +584,228 @@ void accounts::genesis(name user) // Remove this after Golive
...
+
+  uint64_t total = sizes.get("rep.sz"_n.value, "user rep size not set").size;
+  uint64_t current = chunk * chunksize;
+  auto rep_by_rep = rep.get_index<"byrep"_n>();
+  auto ritr = start_val == 0 ? rep_by_rep.begin() : rep_by_rep.lower_bound(start_val);
+  uint64_t count = 0;
+
+  while (ritr != rep_by_rep.end() && count < chunksize) {
+
+    uint64_t rank = (current * 100) / total;
```
Comment (7flash) [inline src/seeds.accounts.cpp:646]: what is this formula?

File: src/seeds.accounts.cpp [hunk 2]
```diff
@@ -542,19 +584,228 @@ void accounts::genesis(name user) // Remove this after Golive
...
-      });
+  // DEBUG REMOVE THIS - THIS IS SO THE FUNCTION DOESN"T RUN AMOK
+  //if (chunk > 10) { 
+  //  return;
+  //}
+
+  uint64_t total = sizes.get("rep.sz"_n.value, "user rep size not set").size;
+  uint64_t current = chunk * chunksize;
+  auto rep_by_rep = rep.get_index<"byrep"_n>();
+  auto ritr = start_val == 0 ? rep_by_rep.begin() : rep_by_rep.lower_bound(start_val);
```
Comment (7flash) [inline src/seeds.accounts.cpp:641]: so we are sorting only this chunk - how do we make sure it's sorted across the chunks?

File: src/seeds.accounts.cpp (hunk 1 above)
Comment (n13) [reply src/seeds.accounts.cpp:646]: same ranking formula as we used before - using current index / total to get a RANK from 0-99

Comment (n13) [reply src/seeds.accounts.cpp:646]: This formula has not changed, and is also tested in the unit tests

File: src/seeds.accounts.cpp (hunk 2 above)
Comment (n13) [reply src/seeds.accounts.cpp:641]: It's sorted by the index - it's always sorted by the system. We only walk over the sorted array and calculate the rank based on the current index vs size of array

**Pull request merged**